app/i18n/index.ts

```diff
@@ -2,82 +2,172 @@
 // See LICENSE.txt for license information.
 
 import moment from 'moment';
+import {Platform} from 'react-native';
+import {getLocales} from 'react-native-localize';
 
 import en from '@assets/i18n/en.json';
 
-export const DEFAULT_LOCALE = 'en';
+const deviceLocale = getLocales()[0].languageCode;
```

Comment:
update this file to include language variants, see v1 and include the new languages that has shipped with MM